### PR TITLE
OCPBUGS-9329: update dynamic plugin info for development mode

### PR DESCRIPTION
**Pull request by rhamilto**
Only show information regarding plugins running locally, and disable the `Enable` button since its not relevant.
 
https://user-images.githubusercontent.com/895728/230417051-dda2a105-3faf-413d-a4b2-6a1259075d08.mov

### CODE REVIEW

**Comment by rhamilto** [comment]
@TheRealJon, I took a stab at plumbing `k8sMode` through.  Mind taking a look?

File: frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx [hunk 1]
```diff
@@ -97,7 +97,40 @@ const ConsolePluginsList: React.FC<ConsolePluginsListType> = ({ obj }) => {
   const [rows, setRows] = React.useState([]);
   const [sortBy, setSortBy] = React.useState<ISortBy>({});
   const pluginColumns = ['name', 'version', 'description', 'status'];
+  const developerMode = window.SERVER_FLAGS.k8sMode === 'off-cluster';
```
Comment (jhadvig) [inline frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx:100]: nit: shouln't it be `developmentMode` ?

File: frontend/packages/console-dynamic-plugin-sdk/README.md [hunk 2]
```diff
@@ -192,8 +192,6 @@ to your local plugin asset server (web server hosting the plugin's generated ass
 Your plugin should start loading automatically upon Console application startup. Inspect the value of
 `window.SERVER_FLAGS.consolePlugins` to see the list of plugins which Console loads upon its startup.
 
-Note running plugins in this way will likely result in incorrect data in the Cluster Dashboard Dynamic Plugins popover.
```
Comment (jhadvig) [inline frontend/packages/console-dynamic-plugin-sdk/README.md:195]: 👍

**Review by jhadvig — COMMENTED** [review]
One nit, otherwise looks good to go 👍

File: frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx (hunk 1 above)
Comment (rhamilto) [reply frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx]: Updated.

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**